src/os/state/basestatemanager.js

```diff
@@ -216,22 +216,26 @@ os.state.BaseStateManager.prototype.deleteStates = function() {
  */
 os.state.BaseStateManager.prototype.getAvailable = function(opt_allVersions) {
   var list = [];
+  var disabledStates = /** @type {Object<string, boolean>} */ (os.settings.get('ex.disabledStates', {}));
```

Comment:
Nit: We generally try to avoid things like `disabled = true` and instead prefer `enabled = false`. Similar to how we enable/disable plugins etc.

Reply:
Heh, I actually had it that way initially and then I switched it to this. I can switch it back.